.github/workflows/ci.yml

```
@@ -0,0 +1,534 @@
+        run: npm install -g vercel@latest
+
+      - name: 🔧 Configure Environment Variables
+        run: |
+          cat > .env.local << EOF
+          NODE_ENV=test
+          E2E_TEST_MODE=true
+          PORT=3000
+          EOF
+
+      - name: 🚀 Start Vercel Dev Server
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:4:34: Double quote to prevent globbing and word splitting

(shellcheck)